app/include/rom.h

```diff
@@ -131,6 +132,8 @@ int ets_printf(const char *format, ...)  __attribute__ ((format (printf, 1, 2)))
 
 int ets_sprintf(char *str, const char *format, ...)  __attribute__ ((format (printf, 2, 3)));
 
+int ets_vsprintf (char *d, const char *s, va_list ap);
```

Comment:
This is not part of the boot ROM. It's provided by the SDK libs. This definition (and the old `ets_sprintf()` one above) should go into `sdk-overrides/include/ets_sys.h`.